client/layout/masterbar/notifications.jsx

```diff
@@ -23,32 +30,51 @@ export default React.createClass( {
 		tooltip: React.PropTypes.string,
 	},
 
+	componentDidMount() {
+		wpcom
+			.req
+			.get( {
+				path: '/notifications/',
+				apiVersion: '1.1'
+			}, { number: 100 } )
+			.then( response => {
+				const { notes } = fromApi( response );
+				console.log( notes[0] );
```

Comment:
there's no reason to keep this here in production, and I don't like messing with setting the debug parameters in the console - I just want the output to show when I'm developing